README.md

```diff
@@ -16,45 +80,114 @@ Generating this manifests is very easy, you just need add the proper dependency
  </dependency>
 ```
 
-We also need to add an annotation to enable dekorate. In this case we will use `@KubernetesApplication` which also gives us access to more Kubernetes specific configuration options.
-Edit the App class and add the indicated annotation.
+We need to enable `Dekorate` by adding an annotation. In this case we will use `@KubernetesApplication` which also gives us access to more Kubernetes specific configuration options.
```

Comment:
To tell to Dekorate what should be populate during the creation of the MANIFEST for Kubernetes, we will add a Java annotation called `@KubernetesApplication` which ... 
Remark: Add here the link to the dekorate documentation please